AbstractAgent.py

```diff
@@ -0,0 +1,11 @@
+import abc
+
+class AbstractAgent(abc.ABC):
+  
+  @abc.abstractmethod
+  def choose(self):
```

Comment:
Before pushing this skeleton, we should add arguments that we expect to at least init. This applies to both classes. When we initialize an abstractAgent, we pass in the algorithm_name, so we should at least address this